### ISSUE TITLE
LazyVim git extras shows 'Not supported spec'

### ISSUE THREAD

**Issue by kevintraver** [polirritmico/telescope-lazy-plugins.nvim#16]
The **_recommended_** config in the LazyVim git extras spec causes a `'Not supported spec'` warning:

```
recommended = {
  ft = { "gitcommit", "gitconfig`", "gitrebase", "gitignore", "gitattributes" },
},
```
https://github.com/LazyVim/LazyVim/blob/24665fc73651970e5a1e20a63d6031bd99b97637/lua/lazyvim/plugins/extras/lang/git.lua?plain=1#L2-L4

It seems to be an issue where none of the cases are matched in `import`

https://github.com/polirritmico/telescope-lazy-plugins.nvim/blob/f588ca7b9a1b50e02d627aade66b666e4e79765f/lua/telescope/_extensions/lazy_plugins/finder/finder.lua?plain=1#L307-L309

**Comment by polirritmico** [comment]
Thanks again for another bug report!

Well, `recommended` is not part of the lazy.nvim spec so that's OK, but it shouldn't matter to the telescope picker.
I've added a simple fix for this. However, if lazyvim or other "distros" add more non-lazy spec fields a more robust solution might be needed. For now I've kept the notification to keep this "on the radar".

Fixed by a8585ad, could you confirm?

**Comment by kevintraver** [comment]
Yes, it is working. No longer see the warning.
